mage_ai/orchestration/pipeline_scheduler_project_platform.py

```diff
@@ -1502,9 +1506,11 @@ def schedule_all():
 
                 concurrency_config = ConcurrencyConfig.load(config=pipeline.concurrency_config)
 
-                lock_key = f'pipeline_schedule_{pipeline_schedule.id}'
```

Comment:
this change is not needed since we don't use this file anymore

Reply:
@wangxiaoyou1993 i removed it